CHANGES.rst

```diff
@@ -4,6 +4,8 @@ unreleased
 Features
 --------
 
+- Deprecated ``pyramid.session.PickleSerializer``.
```

Comment:
Please add the following, but with fully qualified URLs instead of just #s.

```
See: #2709, #3353, #3413
```

Reply:
thanks for catching that. updated. i also signed the contributors file, as i've only signed it in the sideprojects.